pkg/cmd/util/clientcmd/factory_client_access.go

```diff
@@ -267,25 +270,19 @@ func (f *ring0Factory) Printer(mapping *meta.RESTMapping, options kprinters.Prin
 			options.Kind = alias
 		}
 	}
-	return describe.NewHumanReadablePrinter(options), nil
+	return describe.NewHumanReadablePrinter(f.JSONEncoder(), f.Decoder(true), options), nil
 }
 
-func (f *ring0Factory) Pauser(info *resource.Info) (bool, error) {
+func (f *ring0Factory) Pauser(info *resource.Info) ([]byte, error) {
+	// TODO(rebase): compare with origin master, the logic changed completely
 	switch t := info.Object.(type) {
 	case *deployapi.DeploymentConfig:
-		patches := set.CalculatePatches([]*resource.Info{info}, f.JSONEncoder(), func(*resource.Info) (bool, error) {
-			if t.Spec.Paused {
-				return false, nil
-			}
-			t.Spec.Paused = true
-			return true, nil
-		})
-		if len(patches) == 0 {
-			return true, nil
+		if t.Spec.Paused {
+			return nil, errors.New("is already paused")
```

Comment:
shouldn't need to do this check here... if it is already true, the patch should work out to `{}` and result in a 0 status code and an "already paused" message. returning an error here will result in a failed status code, which isn't what we want